### ISSUE TITLE
LB not removed when running as module

### ISSUE THREAD

**Issue by strowi** [kube-hetzner/terraform-hcloud-kube-hetzner#190]
Hi,

i just ran into an issue where the LB is not removed.
This happened when running as a terraform-module and it seems this is related to the `destroy_traefik_loadbalancer` using `${module.path}/..` for the kubeconfig?

when removing the `on_failure = continue` it results in: 
```
│ Error: local-exec provisioner error
│ 
│   with module.rvg.null_resource.destroy_traefik_loadbalancer,
│   on modules/kube-hetzner/main.tf line 127, in resource "null_resource" "destroy_traefik_loadbalancer":
│  127:   provisioner "local-exec" {
│ 
│ Error running command 'kubectl -n kube-system delete service traefik --kubeconfig modules/kube-hetzner/kubeconfig.yaml
│ ': exit status 1. Output: error: stat modules/kube-hetzner/kubeconfig.yaml: no such file or directory

```

regards,
strowi

**Comment by mysticaltech** [comment]
Good find @strowi, good find, don't hesitate to submit a PR.

Otherwise, will fix ASAP. If I remember correctly there are other places in the code where this problem was fixed.

**Comment by mysticaltech** [comment]
Fixed in #191, and fix included in v1.1.9.
